ansible/roles/controller/tasks/deploy.yml

```diff
@@ -16,6 +16,20 @@
     mode: 0777
   become: "{{ logs.dir.become }}"
 
+- name: copy jmxremote password file
+  when: jmx.enabled
+  template:
+    src: "jmxremote.password.j2"
+    dest: "{{ whisk_logs_dir }}/controller{{ groups['controllers'].index(inventory_hostname) }}/jmxremote.password"
```

Comment:
Can we use a config directory here instead (you are creating this in another PR anyway).

Reply:
I think we can leave it as it is, so we both invoker and controller use logs directory. Didn't want to introduce a new config dir in invoker just as a temp storage for jmx config.

Reply:
Hmm I understand not wanting to introduce a new directory but this seems like abusing the purpose of the logs dir. I’d favor using a conf or seperate scratch space also (maybe repurposing the nginx conf dir to be more generic is easier and cleaner).

Reply:
I understand what you mean, but the config won't be stored in the logs directory anyway. It will be moved into the root after we init the component, https://github.com/apache/incubator-openwhisk/pull/2745/files#diff-98fffb6dfa1b0bf01d222972465eeb2eR5. 
We do this kind of _trick_ because in some docker versions, all the files that will be mounted from the host into containers will be given `nouser` and `nogroup` priviledges, which implies that the root user inside of the container won't have permissions to change file if its copied with 0600 or even if the file has 777, the file permissions cannot be changed by root unless you copy it into another file. (Some docker specific behavior that I was not able to change, would be grateful if someone has another workaround) So what I do is copying the file to logs directory with max permission and then move the data with the correct permissions and owners into /root directory.

Reply:
Thanks for the explanation - I was asking something simpler though: could `{{ config_root_dir }}` be the base vs the `{{ whisk_logs_dir }}`?

Reply:
@rabbah {{ whisk_logs_dir }} is now used to move the file

Reply:
As discussed personal: @mhenke1 will change this to the config dir.